Lib/netrc.py

```diff
@@ -23,7 +23,10 @@ class netrc:
     def __init__(self, file=None):
         default_netrc = file is None
         if file is None:
-            file = os.path.join(os.path.expanduser("~"), ".netrc")
+            default_file_name = ".netrc"
```

Comment:
what about inline the `if clause`?

`deafult_file_name = '.netrc' if os.name != 'nt' else '_netrc'`

Reply:
I’m good with that. I’m currently on vacation. Will be back in a week to update PR title and formatting.